crates/host/src/metrics.rs

```diff
@@ -1,24 +1,53 @@
-use wasmcloud_tracing::{Histogram, Meter};
+use wasmcloud_tracing::{Counter, Histogram, Meter, Unit};
 
 /// `HostMetrics` encapsulates the set of metrics emitted by the wasmcloud host
 #[derive(Clone, Debug)]
 #[allow(clippy::module_name_repetitions)]
 pub struct HostMetrics {
     /// Represents the time it took for each handle_rpc_message invocation in nanoseconds.
-    pub wasmcloud_host_handle_rpc_message_duration_ns: Histogram<u64>,
+    pub handle_rpc_message_duration_ns: Histogram<u64>,
+    /// The count of the number of times an actor was invoked.
+    pub actor_invocations: Counter<u64>,
+    /// The count of the number of times an actor invocation resulted in an error.
+    pub actor_errors: Counter<u64>,
+
+    /// The host's ID.
+    // TODO this is actually configured as an InstrumentationScope attribute on the global meter,
+    // but we don't really hve a way of getting at those. We should figure out a way to get at that
+    // information so we don't have to duplicate it here.
+    pub host_id: String,
+    /// The host's lattice ID.
+    // Eventually a host will be able to support multiple lattices, so this will need to either be
+    // removed or metrics will need to be scoped per-lattice.
+    pub lattice_id: String,
 }
 
 impl HostMetrics {
     /// Construct a new [`HostMetrics`] instance for accessing the various wasmcloud host metrics linked to the provided meter.
     #[must_use]
-    pub fn new(meter: &Meter) -> Self {
+    pub fn new(meter: &Meter, host_id: String, lattice_id: String) -> Self {
         let wasmcloud_host_handle_rpc_message_duration_ns = meter
-            .u64_histogram("wasmcloud_host.handle_rpc_message.duration_ns")
+            .u64_histogram("wasmcloud_host.handle_rpc_message.duration")
             .with_description("Duration in nanoseconds each handle_rpc_message operation took")
+            .with_unit(Unit::new("nanoseconds"))
+            .init();
+
+        let actor_invocation_count = meter
+            .u64_counter("wasmcloud_host.actor.invocations")
+            .with_description("Number of actor invocations")
+            .init();
+
+        let actor_error_count = meter
+            .u64_counter("wasmcloud_host.actor.errors")
+            .with_description("Number of actor errors")
```

Comment:
Does it make more sense to call this `actor_invocation_error_count`, since we're tracking errors of actor invocations in particular? "actor error" is a bit vague

Reply:
Also, "actor invocations" seems like it might be confusable -- does it include errored invocations? maybe a more specific name might be helpful, if it's only counting *successful* invocations then maybe that's worth calling out -- unless the expectation (and we should be consistent going forward) is that if "successful" is not stated in the metric, it's implied.

Reply:
> Does it make more sense to call this `actor_invocation_error_count`, since we're tracking errors of actor invocations in particular? "actor error" is a bit vague

It actually ends up being 'actor_error_count' when it makes it to the exporter and Prometheus because the unit of a counter is implicitly a count. The example in the PR description shows that I think

Reply:
ah yeah but the fact that it's an error for an actor *invocation* isn't clear right -- actors could fail to start up as well. But if we think that the only kind of error actors will have is invocation errors, then no need to be more specific.

Reply:
Updated to be `wasmcloud_host.actor.invocation.errors`